app/frontend/src/components/forms/SubmissionsTable.vue

```diff
@@ -562,8 +561,15 @@ export default {
       });
       await this.populateSubmissionsTable();
     },
-    handleSearch(value) {
+    async handleSearch(value) {
+      this.searchEnabled = true;
       this.search = value;
+      if (value === '') {
+        this.searchEnabled = false;
```

Comment:
This searchEnabled is not needed since we know it's disabled if the length of the value is greater than 0.

Reply:
We are using it at the backend.

Reply:
What I meant is that we don't need to create a variable to check whether the search is enabled because we can figure out if search is enabled just by checking if the search string is not empty.

Reply:
Okay👌